app/WikiAsset.js

```diff
@@ -63,7 +63,8 @@ var WikiAsset = declare( Asset, {
 	 * @return {string}
 	 */
 	getUrl: function() {
-		return 'http:' + ( this._wikiUrl || this._api.getDefaultUrl() ) + 'wiki/' + this._filename;
+		var url = ( this._wikiUrl || this._api.getDefaultUrl() ) + 'wiki/' + this._filename;
+		return url.indexOf( 'http' ) === 0 ? url : 'http://' + url;
```

Comment:
Right, I have left in on purpose but actually the cases I had in mind (loading picture from somewhere else than Commons/Wikipedia) are not handled by `WikiAsset` class, so this should indeed be always adding https if no protocol specified. Will fix this